KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/RecordingModule.java

```diff
@@ -84,6 +85,7 @@ public FlavorRecorder(KalturaLiveEntry liveEntry, KalturaLiveAsset liveAsset, Ka
             this.index = index;
             this.isLastChunk = false;
 
+            this.isNewLiveRecordingEnabled = KalturaAPI.getKalturaAPI().isNewRecordingEnabled(liveEntry);
```

Comment:
Can't avoid calling each flavor for the permission. There's no recording entity per entry